modules/profile/manifests/jenkins/rosplugins.pp

```diff
@@ -442,39 +347,19 @@
     require => [ Jenkins::Plugin['bouncycastle-api'] ]
   }
 
-  ::jenkins::plugin { 'workflow-aggregator':
-    version => '2.5',
-    require => [ Jenkins::Plugin['bouncycastle-api'], Jenkins::Plugin['pipeline-build-step'], Jenkins::Plugin['pipeline-input-step'], Jenkins::Plugin['pipeline-milestone-step'], Jenkins::Plugin['pipeline-model-definition'], Jenkins::Plugin['pipeline-stage-step'], Jenkins::Plugin['pipeline-stage-view'], Jenkins::Plugin['workflow-api'], Jenkins::Plugin['workflow-basic-steps'], Jenkins::Plugin['workflow-cps'], Jenkins::Plugin['workflow-cps-global-lib'], Jenkins::Plugin['workflow-durable-task-step'], Jenkins::Plugin['workflow-job'], Jenkins::Plugin['workflow-multibranch'], Jenkins::Plugin['workflow-scm-step'], Jenkins::Plugin['workflow-step-api'], Jenkins::Plugin['workflow-support'] ]
-  }
-
   ::jenkins::plugin { 'workflow-api':
     version => '2.20',
     require => [ Jenkins::Plugin['bouncycastle-api'], Jenkins::Plugin['scm-api'], Jenkins::Plugin['workflow-step-api'] ]
   }
 
-  ::jenkins::plugin { 'workflow-basic-steps':
-    version => '2.6',
-    require => [ Jenkins::Plugin['bouncycastle-api'], Jenkins::Plugin['mailer'], Jenkins::Plugin['structs'], Jenkins::Plugin['workflow-api'], Jenkins::Plugin['workflow-step-api'] ]
-  }
-
   ::jenkins::plugin { 'workflow-cps':
-    version => '2.39',
+    version => '2.40',
     require => [ Jenkins::Plugin['ace-editor'], Jenkins::Plugin['bouncycastle-api'], Jenkins::Plugin['jquery-detached'], Jenkins::Plugin['scm-api'], Jenkins::Plugin['script-security'], Jenkins::Plugin['structs'], Jenkins::Plugin['workflow-api'], Jenkins::Plugin['workflow-scm-step'], Jenkins::Plugin['workflow-step-api'], Jenkins::Plugin['workflow-support'] ]
   }
 
-  ::jenkins::plugin { 'workflow-cps-global-lib':
-    version => '2.8',
-    require => [ Jenkins::Plugin['bouncycastle-api'], Jenkins::Plugin['cloudbees-folder'], Jenkins::Plugin['git-client'], Jenkins::Plugin['git-server'], Jenkins::Plugin['scm-api'], Jenkins::Plugin['script-security'], Jenkins::Plugin['workflow-cps'], Jenkins::Plugin['workflow-scm-step'] ]
-  }
-
-  ::jenkins::plugin { 'workflow-durable-task-step':
-    version => '2.14',
-    require => [ Jenkins::Plugin['bouncycastle-api'], Jenkins::Plugin['durable-task'], Jenkins::Plugin['script-security'], Jenkins::Plugin['workflow-api'], Jenkins::Plugin['workflow-step-api'], Jenkins::Plugin['workflow-support'] ]
-  }
-
   ::jenkins::plugin { 'workflow-job':
-    version => '2.12.2',
-    require => [ Jenkins::Plugin['workflow-api'], Jenkins::Plugin['workflow-step-api'], Jenkins::Plugin['workflow-support'] ]
+    version => '2.11.2',
```

Comment:
Is this an intentional/necessary downgrade?

Reply:
To be perfectly honest, I'm not sure. I haven't figured out the best way to track plugin changes on the production host but the difference between versions doesn't include any described changes that would be issues for us and the downgraded plugin is what's running on the live farm.

This gets us back in sync so I can start testing upgraded plugins across the board so it shouldn't stay downgraded for long.

Reply:
+1